src/clj/game/cards-programs.clj

```diff
@@ -433,6 +433,14 @@
                                     :effect (effect (access-bonus (max 0 target))
                                                     (update! (dissoc (get-card state card) :medium-active)))} card nil))}]}
 
+   "Misdirection"
+   {:abilities [{:cost [:click 2]
+                 :prompt "How many [Credits] to spend to remove that number of tags?"
+                 :choices {:number (req (min (:credit runner) (:tag runner)))}
+                 :msg (msg "spend " target " [Credits] and remove " target " tags")
```

Comment:
`" tags"` could be singular in case the runner wants to spend 2 clicks and 1 credit to remove just one tag.